src/components/AddBook.js

```diff
@@ -0,0 +1,19 @@
+import './addBook.css';
+
+export default function AddBook() {
+  return (
+    <div className="container">
+      <div className="line" />
+      <div className="footer-start">
+        <span className="book-title">
+          ADD NEW BOOK
+        </span>
+        <div className="input-fields">
+          <input className="book-input" placeholder="Book" />
```

Comment:
- [ ] I think the placeholder should be `title` and not Book. What do you think?